File: challenges/codeeval/easy/023/b1nary.py
#python 2.7
solve = ''
for x in range(1,13):
    solve += str(x)
    for y in range(2,13):
        if x*y>=10:
            if x*y>=100:
                solve+=' '+str(x*y)
            else:
                solve+='  '+str(x*y)
        else:
            solve+='   '+str(x*y)
    solve+= '\n'

print solve

File: challenges/codeeval/easy/023/b1nary.c
#include <stdio.h>
int main(void) {
	int x,y,solve;
	for (x=1; x <=12 ; x++){
		printf("%d", x);
		for (y=2; y<=12 ; y++){
			solve = x*y;
			if (solve >= 10 ){
				if (solve >= 100){
					printf(" %d",solve);
				}
				else{
					printf("  %d",solve);
				}
			}
			else {
				printf("   %d",solve);
			}
		}
		printf("\n");
	}
}